go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/0xPolygon/polygon-cli
 
-go 1.22.0
```

Comment:
required for nodekey changes ? pretty big bump.. do we have checks on whether this will break other cli components?

Reply:
we only have checks for loadtest i'm afraid :s